dataquality/core/init.py

```diff
@@ -149,8 +130,8 @@ def init(
         )
         return
 
-    project_name = _init.validate_name(project_name)
-    run_name = _init.validate_name(run_name)
+    project_name = validate_name(project_name)
+    run_name = validate_name(run_name)
```

Comment:
ya don broke the thing
```suggestion
    project_name = validate_name(project_name, assign_random=True)
    run_name = validate_name(run_name, assign_random=True)
```